src/main/scala/com/preprocessing/partitioning/oneDim/Actor.scala

```diff
@@ -0,0 +1,50 @@
+package com.preprocessing.partitioning.oneDim
```

Comment:
As discussed in person, we can later combine this design into the Vertex class and remove the Actor aspect, since we will be having the Akka cluster organize and create actors. 

For now, this will work fine as long as we change the fields containing 'Actors' and 'Mirrors' to Strings or EntityIds (alias for String) based on VertexId and PartitionId instead.